### PR TITLE
pthreads_in_proxy_to_worker_mode

### PR DESCRIPTION
**Pull request by juj**
This makes `-s USE_PTHREADS=1` compatible with `--proxy-to-worker` mode. cc @gouletr.

### CODE REVIEW

**Comment by gouletr** [comment]
Good stuff! I'll try it out asap, thanks!

File: src/proxyWorker.js [hunk 1]
```diff
@@ -460,6 +466,9 @@ onmessage = function onmessage(message) {
       screen.height = Module.canvas.height_ = message.data.height;
       Module.canvas.boundingClientRect = message.data.boundingClientRect;
       document.URL = message.data.URL;
+#if USE_PTHREADS
+      currentScriptUrl = message.data.currentScriptUrl;
```
Comment (kripken) [inline src/proxyWorker.js]: What is this assigning to? I don't see a corresponding `var` here or in higher scopes?

Comment (juj) [inline src/proxyWorker.js]: It intends to replace this https://github.com/kripken/emscripten/blob/master/src/shell.js#L71.

Comment (kripken) [inline src/proxyWorker.js]: Oh ok, cool.

lgtm.

**Pull request merged**